content/blog/2022-02-28-february-22-community-gems.md

```diff
@@ -0,0 +1,182 @@
+---
+title: February '22 Community Gems
+date: 2022-02-28
+description: >
+  A roundup of technical Q&A's from the DVC and CML community. This month:
+  comparing experiments, working with data, working with pipelines, and more.
+descriptionLong: >
+  A roundup of technical Q&A's from the DVC and CML community. This month:
+  comparing experiments, working with data, working with pipelines, and more.
+picture: 2022-01-31/jan-community-gems.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/february-22-community-gems/100132
+tags:
+  - Data Versioning
+  - DVC Remotes
+  - DVC API
+  - DVC Stages
+  - Community
+---
+
+### [Is there a proper way of deleting DVC tracked files from cloud storage?](https://discord.com/channels/485586884165107732/563406153334128681/927618225989111880)
+
+Thanks for the question @fireballpoint1!
+
+You can find the best way to delete files from your cloud storage in
+[our docs](https://dvc.org/doc/command-reference/gc#removing-data-in-remote-storage).
+Make sure you're super careful when deleting data from the cloud because it's an
+irreversible action. Here's an example of a deletion command that will clear out
+your workspace and the cloud storage.
+
+```dvc
+$ dvc gc --workspace --cloud
+```
+
+This option only keeps the files and directories referenced in the workspace and
+it removes everything else, including data in the cloud. By default, this
+command will use the default remote you have set. You can specify a different
+remote storage with the `--remote` option like this.
+
+```dvc
+$ dvc gc --workspace --cloud --remote name_of_remote
+```
+
+### [I'm using DVC experiments for deep learning projects, but I'm running into a problem where the Git index gets corrupted when cache files are above 4 GB. What is the best workaround for this?](https://discord.com/channels/485586884165107732/563406153334128681/928939232033140736)
+
+Great question from @charles.melby-thompson!
+
+This is a known
+[issue with experiments](https://github.com/iterative/dvc/issues/6181) and we
+highly encourage you to comment on and follow this ticket to let us know what
+you need. The reason this happens is because DVC will automatically track `-O`
+outputs with Git internally since it assumes that any outputs that are not
+explicitly part of your `.gitignore` file is part of the experiment state that
+needs to be tracked.
+
+You should be able to explicitly add the `-O` output file(s) to your
+`.gitignore` as a workaround in the meantime.
```

Comment:
I think this dives too deep too quickly for users to understand. Here's my explanation of what I understand from this scenario (needs to be cleaned up a bit):

Experiment files may be tracked by git or dvc. For large files, we generally recommend tracking them with dvc, in which case file size shouldn't be an issue. By default, experiments will track all other files with git. However, git will fail with too much data. If there are files you don't want to track at all (such as large temporary/intermediate files), you can add them to your .gitignore file. See https://github.com/iterative/dvc/issues/6181 for more details and to provide feedback.

Reply:
Thanks for the better explanation on this one!